kt/plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/setupConfigurationsAndCompilations.kt

```diff
@@ -214,13 +218,124 @@ fun Project.setupConfigurationsAndCompilations(godotExtension: GodotExtension, j
                 }
             }
 
+            val checkNativeImageToolAccessible = with(create("checkNativeImageToolAccessible")) {
+                group = "godot-kotlin-jvm"
+                description = "Checks if the GraalVM native image tool is accessible and executable. Needed for GraalVM native-image builds only"
+
+                doLast {
+                    try {
+                        val result = checkToolAccessible(godotExtension.nativeImageToolPath.get())
+                        if (result.exitValue != 0) {
+                            throw IllegalArgumentException("native-image tool not found! Make sure the dx tool is in you PATH variable or set \"dxToolPath\" to the absolute path of the dx tool. Normally the dx tool resides in <android-sdk-root>/build-tools/<version>/dx. For more information visit the docs. Provided path: ${godotExtension.nativeImageToolPath.get()}") //TODO: add url once doc ist hosted
```

Comment:
still some `dx` references in this error message

Reply:
I think I should do an exception as you did for android.